ampersand-model.js

```diff
@@ -20,6 +24,30 @@ var wrapError = function (model, options) {
     };
 };
 
+// recursive function to find the attrs in the serialized object
+function transformForPatch(obj, attrs) {
```

Comment:
It still needs to be named because it's recursive. Is 
```
var transformForPatch = function transformForPatch(obj, attrs) {
``` 
okay? Some folks think it's weird having both.